src/braket/aws/aws_session.py

```diff
@@ -716,7 +716,7 @@ def describe_log_streams(
         self,
         log_group: str,
         log_stream_prefix: str,
-        limit: int = None,
+        limit: Optional[int] = None,
```

Comment:
```suggestion
        limit: int | None = None,
```

Reply:
I use Optional[] if other parameters in this function already use the Optional syntax. I don't think we need to go through all the places where we use that syntax and change it to | None.